packages/api/src/clickhouse/index.ts

```diff
@@ -1,6 +1,6 @@
 import * as fns from 'date-fns';
 import SqlString from 'sqlstring';
-import _ from 'lodash';
+import map from 'lodash/map';
```

Comment:
I think we don't need to do the tree shaking for files in `api/`.

Reply:
I will assume, bundle size is not a concern for server side, as you are not deploying onto a serverless environment. Please correct me if am wrong

Reply:
Correct, its not for the serverless environment